plugins/s3_auth/s3_auth.cc

```diff
@@ -212,14 +213,14 @@ class S3Config
   void
   acquire()
   {
-    ink_atomic_increment(&_ref_count, 1);
+    _ref_count++;
   }
 
   void
   release()
   {
-    TSDebug(PLUGIN_NAME, "ref_count is %d", _ref_count);
-    if (1 >= ink_atomic_decrement(&_ref_count, 1)) {
+    TSDebug(PLUGIN_NAME, "ref_count is %d", (int)_ref_count);
+    if (1 >= _ref_count--) {
```

Comment:
Thought this might be a problems, but it isn't.

Reply:
Which part? I am changing that to a C++ static_cast for @SolidWallOfCode.

Reply:
I was talking about the postfix decrement and when it happens.  It happens before the conditional.